src/http/request.cr

```diff
@@ -33,6 +34,19 @@ class HTTP::Request
     @method == "HEAD"
   end
 
+  def remote_ip(trust_headers=true)
+    if trust_headers
+      client_ips = ips_from("Client-Ip").reverse
+      forwarded_ips = ips_from("X-Forwarded-For").reverse
```

Comment:
As said I've also seen `"X-Real-Ip"` in configs and `X-Fowarded`, `Forwarded` and `"X-Cluster-Client-Ip"` appear to be used values too.